src/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py

```diff
@@ -38,24 +32,23 @@ class ElasticJobParametersDB(ElasticDB):
     def __init__(self, parentLogger=None):
         """Standard Constructor"""
 
-        try:
-            indexPrefix = CSGlobals.getSetup().lower()
+        self.fullname = "WorkloadManagement/ElasticJobParametersDB"
+        self.index_name = self.getCSOption("index_name", "job_parameters")
```

Comment:
I don't see that this `index_name` is documented anywhere.

Reply:
Done.